telegram/ext/conversationhandler.py

```diff
@@ -26,6 +26,13 @@
 from telegram.utils.promise import Promise
 
 
+class ConversationTimeoutContext(object):
```

Comment:
This is an internal class, you probably want to call it `_ConversationTimeoutContext`